integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -121,6 +122,7 @@ class CarbonEnv {
         initialized = true
       }
     }
+    Profiler.initialize(sparkSession.sparkContext)
```

Comment:
Is it the same code in `CarbonSession.scala` not necessary any more after this PR?

Reply:
@kevinjmh After this PR, no need to initialize in CarbonSession.scala. I will remove from CarbonSession.scala